config/foreman-proxy-content.migrations/200818160950-remove_tuning_fact.rb

```diff
@@ -0,0 +1,2 @@
+scenario[:facts].delete('tuning') if scenario[:facts] && scenario[:facts].key?('tuning')
+scenario.delete(:facts) if scenario[:facts] && scenario[:facts] == {}
```

Comment:
Given the repetition of `if scenario[:facts]`, I'd expect a regular `if`
```suggestion
if scenario[:facts]
  scenario[:facts].delete('tuning') if scenario[:facts].key?('tuning')
  scenario.delete(:facts) if scenario[:facts].empty?
end
```

Note this also uses the cleaner `.empty?` method.